armi/cases/tests/test_cases.py

```diff
@@ -103,7 +103,12 @@ def test_summarizeDesign(self):
             )
 
     def test_independentVariables(self):
-        """Ensure that independentVariables added to a case move with it."""
+        """Ensure that independentVariables added to a case move with it.
+
+        .. test:: Write blueprints settings to disk
+            :id: T_ARMI_BP_TO_DB0
+            :tests: R_ARMI_BP_TO_DB
```

Comment:
The statement of this req is:

> The blueprints package shall allow the user to produce a valid blueprints file from an in-memory blueprint object.

So, I think the `saveToStream()` function is probably what we want, and this test isn't so related.